### ISSUE TITLE
Lugares: Búsqueda sensitiva

### ISSUE THREAD

**Issue by mpanero** [mpanero/wt#13]
Habiamos hablado en su monento de que las localidades debían bucarse escribiendo el nombre y presentando opciones...
No veo esa funcionalidad, está hecho ?

**Comment by ruzcat** [comment]
No, siempre se hablo de mostrar una modal con las provincias y dependiendo se cargaban las localidades

**Comment by mpanero** [comment]
OK, pero eso no lo veo tampoco, está listo ?

**Comment by ruzcat** [comment]
Si esta Listo, actualizare todos los códigos fuentes y le enviare la app para que vuelva a validar

**Comment by mpanero** [comment]
En que pantalla está aplicado ? en la creación de una solicitud veo aún el combo
